community.md

```diff
@@ -140,6 +140,7 @@ Key:
 
 | Blog/repo name and description                                          | Author       | Site     | Date        |
 |---------------------------------------------------------------------|--------------|----------|-------------|
+| [.Net Core – OpenFaas – MongoDB](https://marcussmallman.io/2018/11/27/net-core-openfaas-mongodb/) | Marcus Smallman | marcussmallman.io | 28-Nov-2018 |
```

Comment:
The README is very good but the blog is a bit sparse. Can you bring some of they content over like the handler example?

Armhf means Raspberry Pi, so I'm not sure if that's what you meant?

Reply:
I have updated the blog post, I'd be happy to add anything that could have been missed too.

Regarding **armhf**, I have a raspberry pi cluster for which I am deploying my functions too, but am developing them using .net core on my x86 machine. If you're referencing the description made on the architecture used in your mongo function then I meant design architecture :). I have updated the blog post to describe that and I have added a note on the github project stating the currently supported architectures for the function template.